src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
```

Comment:
Magic number '100' should be replaced with a named constant to give it meaning and context. Define it as a private static final variable at the class level.